src/constants/chains.ts

```diff
@@ -5,6 +5,8 @@ import { store } from '@/state/store'
 
 const enabledFlags = preferences.selectors.selectFeatureFlags(store.getState())
 
+export const SECOND_CLASS_CHAINS: readonly KnownChainIds[] = [KnownChainIds.TronMainnet]
```

Comment:
Why not - was gonna say too early, but if that's only used to do `getAccount()` programmatically and do the polling dance, I'm in!